### PR TITLE
Fix SW 5.7 compatibility

### PR DESCRIPTION
**Pull request by felixbrucker**
This PR will ensure the compatibility with SW 5.7.

* Related to https://github.com/pickware/EverythingShopware/issues/50

### CODE REVIEW

**Review by windaishi — APPROVED** [review]

File: Components/PaymentMethods/StripePaymentPreparation.php [hunk 1]
```diff
@@ -128,7 +128,7 @@ protected function getSourceMetadata()
     {
         return [
             'platform_name' => Util::STRIPE_PLATFORM_NAME,
-            'shopware_session_id' => $this->get('SessionID'),
+            'shopware_session_id' => $this->get('sessionid'),
```
Comment (svenmuennich) [inline Components/PaymentMethods/StripePaymentPreparation.php:131]: Does this change work across all supported Shopware versions?

Comment (felixbrucker) [reply Components/PaymentMethods/StripePaymentPreparation.php:131]: Yes, with the old symfony versions the service names were case-insenstive, this changed withe the upgraded symfony dependency in shopware:
> Service identifiers and parameter names are now case sensitive.

**Review by svenmuennich — APPROVED** [review]

File: Views/frontend/checkout/confirm.tpl [hunk 2]
```diff
@@ -58,13 +58,13 @@
                 var stripePublicKey = '{$stripePayment.publicKey|escape:"javascript"}';
                 var stripePaymentSnippets = {
                     error: {
-                        connectionNotSecure: '{stripe_snippet namespace=frontend/plugins/payment/stripe_payment/digital_wallets name=error/connection_not_secure}{/stripe_snippet}',
```
Comment (pajod) [inline Views/frontend/checkout/confirm.tpl:61]: Why this change, when `/Classes/SmartyPlugins.php` was _not_ updated to match what shopware snippet block (`s`) does?
